Cargo.toml

```diff
@@ -11,13 +11,13 @@ categories = ["command-line-utilities", "command-line-interface"]
 edition = "2018"
 
 [features]
-i3 = ["i3ipc"]
+i3 = ["swayipc"]
 
 [dependencies]
 cairo-sys-rs = "0.10"
 css-color-parser = "0.1"
 font-loader = "0.11"
-i3ipc = { version = "0.10", optional = true }
+swayipc = { git = "https://github.com/JayceFayne/swayipc-rs", optional = true }
```

Comment:
For reproducible builds, it would be better if a specific version of `swayipc` was depended on. 

For example, it looks like `swayipc` is about to have a 2.0 release. If that contains breaking changes, it will break this build.